docs/sdk/features.md

```diff
@@ -302,30 +302,39 @@ For instructions on setting up a custom domain, see [Custom Domains](/platform/c
 
 ## Realtime Updates
 
-DevCycle SDKs are capable of being notified in real time that new configuration changes have been made in the DevCycle platform.
+DevCycle SDKs (both client-side and server-side) are capable of being notified in realtime that new configuration changes have been made in the DevCycle platform.
```

Comment:
do we need to clarify that it's both client and server side? why would anyone assume differently when reading this?

Reply:
@ajwootto it feels like there's no harm in being explicit, especially given we didn't start from a place of obviously doing all of the sdks